tests/main/views/test_users.py

```diff
@@ -336,61 +322,48 @@ def test_can_not_post_an_admin_ccs_user(self):
                     'name': 'joe bloggs'}}),
             content_type='application/json')
 
-        assert_equal(response.status_code, 400)
+        assert response.status_code == 400
         error = json.loads(response.get_data())['error']
-        assert_in("'admin-ccs' is not one of", error)
-
-    def test_can_post_a_supplier_user(self):
-        with self.app.app_context():
-            db.session.add(
-                Supplier(supplier_id=1, name=u"Supplier 1")
-            )
-            db.session.commit()
+        assert "'admin-ccs' is not one of" in error
 
+    def test_can_post_a_supplier_user(self, supplier_basic):
```

Comment:
[It's a fixture, Kev](https://github.com/alphagov/digitalmarketplace-api/blob/a1f9f4dce3d93d53f11bcce575329ff8e635d525/tests/conftest.py#L390).  Define it in `conftest.py` and then pass it in as a parameter to any method.

Amazing, no?

I _could_ have used the setup_dummy method again, but it felt like a lot of overhead when we just wanted something simple (plus there's that whole faff of having to add 2 of them because [the first supplier added by `setup_dummy_suppliers()` has an id of 0](https://github.com/alphagov/digitalmarketplace-api/blob/a1f9f4dce3d93d53f11bcce575329ff8e635d525/tests/helpers.py#L114), which is rejected when [trying to create a supplier user](https://github.com/alphagov/digitalmarketplace-api/blob/master/app/main/views/users.py#L294)).

If you think it's not simple though, I can change it to use the `setup_dummy_suppliers()` method.